### ISSUE TITLE
export issue status (ff -> GitHub)

### ISSUE THREAD

**Issue by cblp** [ff-notes/ff#59]
close GitHub issue that is closed in ff

**Comment by cblp** [comment]
we don't need it

**Comment by cblp** [comment]
may be useful

**Comment by willbasky** [comment]
> closed in ff

when Status is `Archived`?

**Comment by cblp** [comment]
fixed description

**Comment by willbasky** [comment]
Should it be automagically within `ff done ...` command or with separate one?

**Comment by cblp** [comment]
Yes, extending the behavior of the `done` command and corresponding button in UI. Now they don't and can't do anything.
